am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/importer/DefaultPermissionIntegrationTest.java

```diff
@@ -84,11 +84,11 @@ void shouldRemoveAllEntriesFromTablesWhenValuesExist() {
 
         service.truncateDefaultPermissionsForService(SERVICE_NAME, RESOURCE_TYPE);
 
-        assertThat(databaseHelper.countDefaultPermissions(SERVICE_NAME, RESOURCE_TYPE, RESOURCE_NAME, ATTRIBUTE,
-            ROLE_NAME, Permissions.sumOf(READ_PERMISSION))).isEqualTo(0);
+        assertThat(databaseHelper.countDefaultPermissions(SERVICE_NAME, RESOURCE_TYPE, RESOURCE_NAME,
```

Comment:
Sadly I did not find a way to take JsonPointer as a straight argument. It is not a problem in main repositories because they take wrapper objects that expose `attributeAsString`.